app/uk/gov/hmrc/rasapi/repository/RasFileRepository.scala

```diff
@@ -53,7 +53,7 @@ class RasFilesRepository @Inject()(
   addAllTTLs(gridFSG)
 
   def saveFile(userId:String, envelopeId: String, filePath: Path, fileId: String): Future[ResultsFile] = {
-    Logger.info("[RasFileRepository] Starting to save file")
+    Logger.info("[RasFileRepository][saveFile] Starting to save file")
     val fileToSave = DefaultFileToSave(s"${fileId}", Some(contentType),
       metadata = BSONDocument("envelopeId" -> envelopeId, "fileId" -> fileId, "userId" -> userId))
 
```

Comment:
As a general, in this file they use 'logger' which is one that comes from reactivemongo, I'm unsure whether that's what we want to use. To be safe I would swap over to the standard one we use everywhere else.